cinema/admin.py

```diff
@@ -1,19 +1,44 @@
 from django.contrib import admin
+from .models import (Genre,
+                     Actor,
+                     Movie,
+                     CinemaHall,
+                     MovieSession,
+                     Order,
+                     Ticket)
 
-from .models import (
-    CinemaHall,
-    Genre,
-    Actor,
-    Movie,
-    MovieSession,
-    Order,
-    Ticket,
-)
-
-admin.site.register(CinemaHall)
-admin.site.register(Genre)
-admin.site.register(Actor)
-admin.site.register(Movie)
-admin.site.register(MovieSession)
-admin.site.register(Order)
-admin.site.register(Ticket)
+
+@admin.register(Genre)
+class GenreAdmin(admin.ModelAdmin):
+    list_display = ["id", "name"]
+
+
+@admin.register(Actor)
+class ActorAdmin(admin.ModelAdmin):
+    list_display = ["id", "first_name", "last_name"]
+
+
+@admin.register(Movie)
+class MovieAdmin(admin.ModelAdmin):
+    list_display = ["id", "title", "duration"]
+    filter_horizontal = ["genres", "actors"]
+
+
+@admin.register(CinemaHall)
+class CinemaHallAdmin(admin.ModelAdmin):
+    list_display = ["id", "name", "rows", "seats_in_row", "capacity"]
```

Comment:
The `capacity` field in `list_display` might not exist in the `CinemaHall` model. Ensure that this field is defined in the model, or remove it from `list_display` if it does not exist.